src/App/App.js

```diff
@@ -20,6 +22,8 @@ function App() {
       <Container fluid className="main-container">
         <Switch>
           <Route exact path="/" component={HomePage} />
+          <Route exact path="/media" component={MediaPage} />
```

Comment:
Media page has been removed, the home page now supports Feed which merges Instagram, Twitter and soon Facebook posts. (As a manual input of an URL)